lib/kcerts/ssh.go

```diff
@@ -0,0 +1,116 @@
+package kcerts
+
+import (
+	"bufio"
+	"bytes"
+	"crypto/rand"
+	"crypto/rsa"
+	"fmt"
+	"github.com/mitchellh/go-homedir"
+	"golang.org/x/crypto/ssh"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"strings"
+	"time"
+)
+
+const CAPrefix = "@cert-authority"
+const SSHDir = ".ssh"
+const KnownHosts = "known_hosts"
```

Comment:
The reason why I'm fine with how it is now is because since each go test is essentially an isolate we can just change the homedir in each test to some tmp file and call it good.  That way it mimics somewhat its real case. 

If you think eventually kcerts is going to move to more of an ssh/ecdsa/rsa utility package I would definitely refactor into a service later on.

Reply:
Especially if we end up isolating ALL ssh dependencies into $HOME/.enkit (which is a cool idea). That way the end user never has to deal with local .ssh screwing up with other projects.